test/integration-tests/IntegrationTests.Helpers/TestHelper.cs

```diff
@@ -103,6 +103,50 @@ public async Task<Container> StartContainerAsync(int traceAgentPort, int webPort
         return new Container(container);
     }
 
+    public async Task<Container> StartContainerForMetricsAsync(int webPort, int prometheusPort)
```

Comment:
Consider to extract one, common method to start containers. It could us settings parameters, similar to `public ProcessResult RunTestApplicationAndWaitForExit(TestSettings testSettings)`

We avoid duplication code with this approach.